src/main/java/com/salesforce/dataloader/action/visitor/BulkApiVisitorUtil.java

```diff
@@ -80,6 +84,29 @@
         } catch (ParameterLoadException e) {
             throw new RuntimeException("Failed to initialize check status interval", e);
         }
+        
+        /*
+         * ======== Start code block to support PK chunking
+         *
+        this.enablePKChunking = ctl.getConfig().getBoolean(Config.ENABLE_BULK_QUERY_PK_CHUNKING);
```

Comment:
The value of enablePKChunking was set in line 73, if you are assigning it in this constructor can we omit the assignment in line 73.

Reply:
This is in the commented out code. That's why I had to set it to false on line 73.